.github/workflows/query-engine-driver-adapters.yml

```diff
@@ -68,11 +70,19 @@ jobs:
       - name: 'Login to Docker Hub'
         uses: docker/login-action@v3
         continue-on-error: true
-        if: "${{ secrets.DOCKERHUB_USERNAME != '' && secrets.DOCKERHUB_TOKEN != '' }}"
```

Comment:
Accidental delete I think? That was just recently added to avoid a problem: https://github.com/prisma/prisma-engines/commit/51d8349124b96b4c636526990eba13a691d553a4